common/include/pcl/experimental/executor/blocking.cpp

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2014-, Open Perception, Inc.
+ *  Author(s): Shrijit Singh <shrijitsingh99@gmail.com>
+ *
+ */
+#include <pcl/experimental/executor/property.h>
```

Comment:
You don't need property.h, just the header related to blocking.

Reply:
Fixed in v0.3

Reply:
Done